src/GitTools.IssueTrackers/IssueTrackers/Jira/JiraIssueTracker.cs

```diff
@@ -1,32 +1,34 @@
 ﻿namespace GitTools.IssueTrackers.Jira
 {
-    using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading.Tasks;
-    using Atlassian.Jira;
+    using Jira = Atlassian.Jira;
     using Logging;
-    using Issue = IssueTrackers.Issue;
-    using IssueType = IssueTrackers.IssueType;
-    using Version = IssueTrackers.Version;
 
-    public class JiraIssueTracker : IssueTrackerBase
+    public class JiraIssueTracker : IIssueTracker
     {
+        private static readonly HashSet<string> KnownClosedStatuses = new HashSet<string>(new[] { "resolved", "closed", "done" });
         private static readonly ILog Log = LogProvider.GetCurrentClassLogger();
 
-        private static readonly HashSet<string> KnownClosedStatuses = new HashSet<string>(new [] { "resolved", "closed", "done" }); 
+        private readonly string _server;
+        private readonly AuthenticationContext _authenticationInfo;
 
-        public JiraIssueTracker(IIssueTrackerContext issueTrackerContext)
-            : base(issueTrackerContext)
+        public JiraIssueTracker(string server, string project, AuthenticationContext authenticationInfo)
         {
+            // TODO Project?
```

Comment:
I am passing it through, just not sure how to apply it (it wasn't being applied before these changes)